### PR TITLE
Cardano Signer deployment

### PR DESCRIPTION
**Pull request by Scitz0**
## Description
A tool by Martin to:
- Sign data with a Cardano-Secret-Key
- Verify data with a Cardano-Public-Key
- Generate CIP-8 & CIP-36 data

For full details, visit.
https://github.com/gitmachtl/cardano-signer

## How has this been tested?
It is tested on Ubuntu x64 system to download the correct binary. It needs to be tested on various systems, Linux, Mac, etc. Currently, there is no ARM binary available, the prereqs script might need to be tweaked later.

### CODE REVIEW

File: docs/basics.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ Install pre-requisites for building cardano node and using CNTools
 -c    Install/Upgrade and build CNCLI with RUST
 -w    Install/Upgrade Vacuumlabs cardano-hw-cli for hardware wallet support
 -o    Install/Upgrade Ogmios Server binary
+-x    Install/Upgrade Cardano Signer (https://github.com/gitmachtl/cardano-signer)
```
Comment (rdlrt) [inline docs/basics.md:50]: IMO - we shouldnt have links on `-h`, as we dont for cardano-hw-cli/ogmios/cncli too.

Instead, we could potentially add this to [Build overview](https://cardano-community.github.io/guild-operators/build/) and have a brief couple of liners about each additional component, linking to their repos - ofcourse, if folks who've built the tools dont have a concern

File: scripts/cnode-helper-scripts/prereqs.sh [hunk 2]
```diff
@@ -73,14 +74,15 @@ Install pre-requisites for building cardano node and using CNTools
 -c    Install/Upgrade and build CNCLI with RUST
 -w    Install/Upgrade Vacuumlabs cardano-hw-cli for hardware wallet support
 -o    Install/Upgrade Ogmios Server binary
+-x    Install/Upgrade Cardano Signer (https://github.com/gitmachtl/cardano-signer)
```
Comment (rdlrt) [inline scripts/cnode-helper-scripts/prereqs.sh:77]: (same as above)

File: docs/basics.md (hunk 1 above)
Comment (rdlrt) [reply docs/basics.md]: Apologies for delay in adding above mentioned Build Overview, done now

**Review by rdlrt — APPROVED** [review]
LGTM - but maybe go through `docs/build.md` to proof-read (added few mins ago) before merging

**Review by gufmar — APPROVED** [review]
nice addition

**Pull request merged**